Integrations/integration-fireeye.yml

```diff
@@ -468,3 +516,60 @@ script:
       description: FireEye application ID
     - contextPath: FFireEyeAX.Sensors.Profiles.Applications.-name
       description: FireEye application name
+    description: commands.server.feConfig.description
+  - name: fe-submit-url
+    arguments:
+    - name: analysistype
+      required: true
+      description: 'Specify live or sandbox analysis mode. 0—Live, analyze suspected
+        malware objects live within the MAS Multivector Virtual Execution (MVX) analysis
+        engine. 1—Sandbox, analyze suspected malware objects in a closed, protected
+        environment. Example: analysisType=live'
+    - name: profiles
+      required: true
+      description: Select the MAS profile to use for analysis. To determine the available
+        profiles, use the configuration command
```

Comment:
Please write and add an example of profiles so it would be clear that it should be comma separated.

Reply:
Included example for clarification